### ISSUE TITLE
 abstract-syntax-tree is broken due to meriyah

### ISSUE THREAD

**Issue by aminya** [buxlabs/amd-to-es6#125]
```
❯ amdtoes6 --src=./src --replace
C:\a\node_modules\meriyah\dist\meriyah.umd.js:188
      throw new ParseError(parser.index, parser.line, parser.column, type, ...params);
      ^
```

**Comment by emilos** [comment]
hey @aminya thanks for the report. Could you please provide more information? Could you please share the code that you're trying to transform?

edit: I've updated meriyah and released a new version of the lib (0.16.0) - maybe try this one too?

**Comment by emilos** [comment]
The library got updated, so I guess the issue has been resolved, but please reopen if that's not the case.
